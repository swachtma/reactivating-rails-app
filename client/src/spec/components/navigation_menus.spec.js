/* global expect, jest */
import React from 'react'
import { shallow } from 'enzyme'

import configEnzyme from '../../test_helper'
import NavigationMenus from '../../components/navigation_menus'

let w = shallow(<NavigationMenus />)

describe('<NavigationMenus />', () => {
  it('renders according to snapshot', () => expect(w).toMatchSnapshot())
})
