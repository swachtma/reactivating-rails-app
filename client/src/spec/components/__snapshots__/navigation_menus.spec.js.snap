// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationMenus /> renders according to snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavigationMenus />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": "sidebar",
    "nodeType": "class",
    "props": Object {},
    "ref": Object {
      "current": null,
    },
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": "sidebar",
      "nodeType": "class",
      "props": Object {},
      "ref": Object {
        "current": null,
      },
      "rendered": null,
      "type": [Function],
    },
    Object {
      "instance": null,
      "key": "menubar",
      "nodeType": "function",
      "props": Object {
        "handleSidebarVisibility": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
